scripts/ci/wasm_unpack.js

```diff
@@ -0,0 +1,52 @@
+7/*\
+|*|
+|*|  Base64 / binary data / UTF-8 strings utilities (#1)
+|*|
+|*|  https://developer.mozilla.org/en-US/docs/Web/API/WindowBase64/Base64_encoding_and_decoding
+|*|
+|*|  Author: madmurphy
```

Comment:
I think the conclusion was "Public Domain", when I had it in the old PR, but we should recheck.

Reply:
Just checked again, it's explained here: https://developer.mozilla.org/en-US/docs/MDN/About#copyrights_and_licenses

Reply:
Actually "Since the launch of the new Yari MDN platform on December 14 2020, there is currently no way to determine which one you need. We are working on this and will update this content soon." is a bit unfortunate, but I think when I last checked it was earlier than December 14 2020 and it was clear that "Public Domain" applied in this case.